src/EnergyPlus/PackagedTerminalHeatPump.cc

```diff
@@ -5046,6 +5066,25 @@ namespace PackagedTerminalHeatPump {
 		PTUnit( PTUnitNum ).LatHeatEnergy = PTUnit( PTUnitNum ).LatHeatEnergyRate * ReportingConstant;
 		PTUnit( PTUnitNum ).ElecConsumption = PTUnit( PTUnitNum ).ElecPower * ReportingConstant;
 
+		if ( PTUnit( PTUnitNum ).FirstPass ) { // reset sizing flags so other zone equipment can size normally
+
+			if( !SysSizingCalc ) {
+
+				if ( CurZoneEqNum > 0 ) {
```

Comment:
I agree. I was just thinking about this last night and would rather use a function call that resets everything. I've only used this in 2 places so far and think it needs to be used anywhere these logical/value pairs are used in a parent (e.g., fan coil).